lib/form_data.js

```diff
@@ -31,7 +31,7 @@ function FormData() {
   this._valueLength = 0;
   this._valuesToMeasure = [];
 
-  CombinedStream.call(this);
+  Object.assign(this, CombinedStream.create(options));
```

Comment:
As I can see Object.assign is failing builds do you want me to use util._extend, or write a util function to do it?